pants.ci.toml

```diff
@@ -26,3 +26,10 @@ ci_env_variables = [
 
 [buildsense]
 enable = true
+
+[subprocess-environment]
+env_vars.add = [
+  # Works around bad `-arch arm64` flag embedded in Xcode 12.x Python interpreters on intel
+  # machines. See: https://github.com/giampaolo/psutil/issues/1832
+  "ARCHFLAGS",
```

Comment:
I suppose that this is never needed inside of a test (and thus doesn't need to be in `[test]`), because Pants is no longer actually bootstrapping itself from source inside of tests (and instead loads from the python path).

Reply:
I think that's right.